instances/UBERONParcellation/ansiformLobuleCrusII.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/ansiformLobuleCrusII",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+    "definition": "'Ansiform lobule crus II' is an ansiform lobule and hemisphere part of cerebellar posterior lobe.",
+    "description": "",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0104942",
+    "knowledgeSpaceLink": "https://knowledge-space.org/wiki/NLXANAT:20081209#ansiform-lobule-crus-ii",
+    "name": "ansiform lobule crus II",
+    "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/UBERON_0005977",
+    "synonym": [
+        "Lobulus Semilunaris Inferior",
+        "Lobulus Ansiform Crus II",
+        "Inferior Semilunar Lobule",
+        "Semilunar lobule-2 (inferior)",
+        "Inferior semilunar lobule of cerebellum",
+        "Posterior inferior lobule",
+        "second crus of ansiform lobule of cerebellum [hVIIa]",
+        "crus secundum lobuli ansiformis cerebelli [hVII A]",
+        "hemispheric lobule VIIBi",
+        "crus II of the ansiform lobule (HVII)",
+        "lobulus semilunaris inferior cerebelli",
+        "lobulus semilunaris caudalis",
+        "ansiform lobe of the cerebellum, crus 2",
+        "crus 2",
+        "Cb-Crus II",
+        "crus II",
+        "lobulus semilunaris cranialis",
+        "crus II (cerebelli)",
+        "lobulus posterior inferior"
+    ]
+}
```

Comment:
Please add a newline at the end of the JSONLD file.

Reply:
Done.